includes/TripalImporter/BibtexImporter.inc

```diff
@@ -0,0 +1,108 @@
+<?php
+
+class BibtexImporter extends TripalImporter {
+
+  /**
+   * The name of this loader.  This name will be presented to the site
+   * user.
+   */
+  public static $name = 'Chado bibtex Loader';
+
+  /**
+   * The machine name for this loader. This name will be used to construct
+   * the URL for the loader.
+   */
+  public static $machine_name = 'chado_bibtex_loader';
+
+  /**
+   * A brief description for this loader.  This description will be
+   * presented to the site user.
+   */
+  public static $description = 'Load bibtex formatted publications into Chado';
+
+  /**
+   * An array containing the extensions of allowed file types.
+   */
+  public static $file_types = [
+    '.txt',
+    '.tex',
+    'bib',
+  ];
+
+  /**
+   * Allow, but dont require, an analysis.
+   */
+  public static $use_analysis = TRUE;
+
```

Comment:
I would set `$use_analysis` to `FALSE` since we don't do anything with it. It feels misleading to include it ;-)